docs/HACKING.md

```diff
@@ -93,6 +93,23 @@ export OPENSHIFT_CI=false
 # on AWS only:
 export AWS_SHARED_CREDENTIALS_FILE=<path to aws credentials file>
 ```
+
+In addition, just for **BYOH** use case with **platform=none** UPI cluster export the `WINDOWS_INSTANCES_DATA` 
```

Comment:
Shouldn't we explain how the instances will/should be brought up when running this scenario locally?

Reply:
Makes sense, will adjust.